src/op_handlers/opcode_decommit.rs

```diff
@@ -52,11 +52,7 @@ pub fn opcode_decommit(
 
     let heap = vm.heaps.get_mut(id).ok_or(HeapError::StoreOutOfBounds)?;
 
-    let mut address = 0;
-    for value in code.iter() {
-        heap.store(address, *value);
-        address += 32;
-    }
```

Comment:
Seeing it starts with address 0, maybe we can just assign `code` to the new heap instead. This way we avoid all the copying and just move the whole vector.

Reply:
Good idea!

Reply:
I actually don't know if there is an easy way of doing that, since `code` is of type `Vec<U256>` but `heap` is of type `Vec<u8>`.
I'll keep the code as is, but if you happen to know how to do that transformation it could be a good optimization.

Reply:
I am not saying to make this change now, but something to consider for a future pr: we could easily retrieve the code as a `Vec<u8>` by adding a new function to the `Storage` trait that decommits and returns it in that format. The reason it's easy is that in the current `zksync-era` storage, bytecodes are originally stored as `Vec<u8>`, and we are actually converting them to Vec<U256> (see [here](https://github.com/lambdaclass/zksync-era/blob/era_vm_integration_v2/core/lib/multivm/src/versions/era_vm/vm.rs#L724-L747)).

Reply:
Yep, if we are able to modify that trait (we are not following any API), then thats totally an option.